src/constants/url.ts

```diff
@@ -37,10 +37,12 @@ export const getServerInfo = () => {
 
     const existingAppId = LocalStorageUtils.getValue<string>(LocalStorageConstants.configAppId);
     const existingServerUrl = LocalStorageUtils.getValue<string>(LocalStorageConstants.configServerURL);
-    // since we don't have official app_id for staging,
     // we will use the red server with app_id=62019 for the staging-p2p.deriv.com for now
-    // to fix the login issue
-    if (origin === URLConstants.derivP2pStaging && (!existingAppId || !existingServerUrl)) {
+    // for p2p.deriv.com, we will use the green server with app_id=61859
+    if (
+        (origin === URLConstants.derivP2pStaging || origin === URLConstants.derivP2pProduction) &&
```

Comment:
i don't think blue and green environments can be access in staging since it doesn't have an official app id. Can you please check?

Reply:
sure let me check

Reply:
yup we can't use green or blue for staging, just red, but this should be ok, because below itll map p2pProduction to green, staging to red